bookshelf/app.yaml.dist

```diff
@@ -0,0 +1,12 @@
+runtime: custom
+vm: true
+
+manual_scaling:
```

Comment:
Also I'm going to rename `app.yaml.dist` to `app.yaml` which is simpler. As long as we have `app.yaml` in the top level `.gitignore`, it won't be a problem.